### PR TITLE
Show dues reminders in Nova

### PR DESCRIPTION
**Pull request by kberzinch**

### CODE REVIEW

File: app/Jobs/CreateDuesPaymentDueNotificationInNova.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    public function handle()
+    {
+        if ($this->user->dues()->pending()->count() > 0 &&
+            $this->user->hasPermissionTo('access-nova') &&
+            0 === Notification::where('notifiable_type', '=', User::class)
+                ->where('notifiable_id', '=', $this->user->id)
+                ->where('type', DuesPaymentDue::class)
+                ->where('created_at', '>', now()->subMonths(3))
+                ->count()
+        ) {
```
Comment (evan10s) [inline app/Jobs/CreateDuesPaymentDueNotificationInNova.php:52]: Might be a decent candidate for a query scope on the User class?

File: app/Notifications/Nova/DuesAreLive.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+     * @var string|null
+     */
+    public $message = 'Dues are live! Take a moment to pay online now.';
+
+    /**
+     * The text used for the call-to-action button label.
+     *
+     * @var string
+     */
+    public $actionText = 'Get started';
```
Comment (evan10s) [inline app/Notifications/Nova/DuesAreLive.php:37]: I'd prefer action-oriented text on this button as well, but this `Dues are live` notification type seems like overlaps a lot with the `Dues payment due` notification.  I almost feel like we should only show one or the other.

Comment (kberzinch) [reply app/Notifications/Nova/DuesAreLive.php:37]: `DuesAreLive` means you need to create a `DuesTransaction`, `DuesPaymentDue` means you need to complete a `Payment`. The prune job will delete a `DuesAreLive` notification once a `DuesTransaction` is created.

File: app/Jobs/CreateDuesPaymentDueNotificationInNova.php (hunk 1 above)
Comment (kberzinch) [reply app/Jobs/CreateDuesPaymentDueNotificationInNova.php:52]: I don't think a query scope is quite the right refactor, but I will make it an Eloquent relationship.

File: app/Notifications/Nova/DuesAreLive.php (hunk 2 above)
Comment (evan10s) [reply app/Notifications/Nova/DuesAreLive.php:37]: That all makes sense now, might be good to leave a quick note with the intended distinction in a comment on one/both of these classes for posterity

Comment (kberzinch) [reply app/Notifications/Nova/DuesAreLive.php:42]: Added comments to both classes.

**Review by evan10s — APPROVED** [review]

**Pull request merged**